composer.json

```diff
@@ -322,6 +322,7 @@
                 "Invalid argument supplied for foreach SettingsForm->prepareHeadersDisplay (https://www.drupal.org/project/access_unpublished/issues/3252499)": "https://www.drupal.org/files/issues/2021-12-03/3252499-warning-invalid-argument-2.patch"
             },
             "drupal/core": {
+                "Disable body-level scrolling when a dialog is open as a modal #2707291" : "https://www.drupal.org/files/issues/2022-01-30/2707291-62_0.patch",
```

Comment:
FYI we recently adopted a policy at Lullabot of committing patches to the repository: https://architecture.lullabot.com/adr/20220429-composer-patch-files/

It's something to consider! cc @tom-fleming @weitzman .

Reply:
AFAIK, It is 100% safe to reference patches from drupal.org. They cant be changed. Its less safe to reference Gitlab patches as those can change.